api/server_test.go

```diff
@@ -0,0 +1,204 @@
+package api_test
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/gorilla/websocket"
+	"github.com/onflow/flow-evm-gateway/api"
+	"github.com/onflow/flow-evm-gateway/storage"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+)
+
+var requests = []string{
```

Comment:
it would be better to extract this to fixture files, you can create `request.json` and then use embed to embed it.

Reply:
Nice idea :+1: 
Fixed in https://github.com/onflow/flow-evm-gateway/pull/11/commits/e893a6f77ea2e19f29f99ff711837033fd86325b